### PR TITLE
[VL] Minor improvements for CI stale bot

### PR DESCRIPTION
**Pull request by zhztheplayer**
1. ~~Leave comments on issues as well as on PRs.~~ Disable auto-close on issues.
2. Note user to feel free to reopen.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -25,7 +25,9 @@ jobs:
...
         with:
           stale-pr-message: 'This PR is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
-          close-pr-message: 'This PR was closed because it has been stalled for 10 days with no activity.'
-          days-before-pr-stale: 45
-          days-before-pr-close: 10
+          close-pr-message: 'This PR was auto-closed because it has been stalled for 10 days with no activity. Please feel free to reopen if it's still valid. Thanks.'
+          stale-issue-message: 'This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-issue-message: 'This issue was auto-closed because it has been stalled for 10 days with no activity. Please feel free to reopen if it's still valid. Thanks.'
+          days-before-stale: 45
+          days-before-close: 10
```
Comment (zhouyuan) [inline .github/workflows/stale.yml:32]: I actually prefer do not close issues

Comment (zhztheplayer) [reply .github/workflows/stale.yml:32]: `actions/stale` closes issues by default... So let's disable it in this PR?

Comment (zhztheplayer) [reply .github/workflows/stale.yml:32]: e.g. [this one](https://github.com/oap-project/gluten/issues/3235#event-11090616372)

**Review by zhouyuan — APPROVED** [review]
👍

**Pull request merged**